src/lib.rs

```diff
@@ -9,67 +9,135 @@ use std::cmp::min;
 pub mod modern_skin;
 
 /// Creates a Smash Ultimate Minecraft Steve inspired render from the given Minecraft skin texture.
-pub fn create_render(skin_texture: &RgbaImage) -> RgbaImage {
+fn create_render_of(skin_texture: &RgbaImage, is_alex: bool) -> RgbaImage {
```

Comment:
The doc comments should be moved to the public functions. I would prefer `create_render_inner` instead of `create_render_of`. If you can think of a better name, that's also fine.